src/main/clojure/clara/rules/durability/fressian.clj

```diff
@@ -235,6 +235,16 @@
                 (read [_ rdr tag component-count]
                   (read-with-meta rdr #(apply list %))))}}
 
+   "clj/emptylist"
+   {:class clojure.lang.PersistentList$EmptyList
+    :writer (reify WriteHandler
+              (write [_ w o]
+                (write-with-meta w "clj/emptylist" o)))
```

Comment:
Since this type is, by definition, empty, we don't actually need to write out or read in the list itself - we could just write out the metadata if present (or null if absent).  See the implementation of write-with-meta and read-with-meta.